packages/metascraper-helpers/index.js

```diff
@@ -87,7 +89,7 @@ const normalizeUrl = (baseUrl, relativePath, opts) => {
 
 const removeByPrefix = flow([value => value.replace(REGEX_BY, ''), trim])
 
-const createTitle = flow([condenseWhitespace, smartquotes])
+const createTitle = flow([entities.decode, condenseWhitespace, smartquotes])
```

Comment:
Can you explain why this is necessary?

All the HTML content is [decoded](https://github.com/microlinkhq/metascraper/blob/master/packages/metascraper/src/load-html.js#L8) at the first moment, so you should not be worried about this at this level

Reply:
The JSON-LD is JSON encoded data within a `<script>` tag within the HTML.  A few examples in the integration tests had HTML entities in the title. I don't see any cases where you'd specifically *want* to have `&amp;` in a title.

Reply:
but are these encoded value not decoded on the load stage?

Reply:
The load stage only JSON.parses the data. Looping through the hash and HTML-decoding fields without knowing what the fields are seems brittle to me.

Reply:
> The load stage only JSON.parses the data. Looping through the hash and HTML-decoding fields without knowing what the fields are seems brittle to me.

I don't understand this 🤔 the `.loadHTML` is the first thing done. Then, JSON-LD should be extracted from the normalized HTML obtained there.

Reply:
Since JSON-LD is embedded inside a `<script>` tag it is not HTML decoded. It's HTML that contains JSON with a string containing HTML entities.

I'm not sure does the JSON-LD spec say anything about HTML encoding data, but there were several examples in the integration tests which resulted in HTML entities being in the output. As it seems to be relatively commonplace, the HTML unencoding is justified.  It could be limited to the JSON-LD case, but I don't see any reasonable cases where it would break anything.

An example is `silicon-beat/input.html` which contains:

```
   <script type="application/ld+json">
   {"@context":"http:\/\/schema.org",
   "@type":"NewsArticle",
   "headline":"VC: Time to &#8216;come out as a woman&#8217;",
...
   }
   </script>
```

Reply:
OK, now I followed you 🙂

In that case, decode script entitiy should be JSON-LD helper responsibility!